rackcity/views/rack_views.py

```diff
@@ -76,7 +76,31 @@ def rack_create(request):
     """
     Create racks within specified range.
     """
-    return JsonResponse({}, status=HTTPStatus.INTERNAL_SERVER_ERROR)
+    range_serializer = RackRangeSerializer(data=request.data)
+    if not range_serializer.is_valid():
+        return JsonResponse(
+            {"failure_message": "Invalid keys or values in request."},
```

Comment:
I think you can do `{"failure_message": str(range_serializer.errors)}` here can't you?

Reply:
so true. fixing this